src/components/tabs/_tabs.scss

```diff
@@ -90,57 +91,78 @@
           padding-bottom: govuk-spacing(2);
         }
 
-        @include govuk-if-ie8 {
-          // IE8 doesn't support `box-shadow` so add an actual border that is
-          // modified to indicate state.
-          border-top: 4px solid transparent;
+        // Since borders are joined with a  45 degree angle bevel 'mitre joint',
+        // we can't set the top border color.
+        //
+        // +-------------+
+        // | \=========/ |
+        // |   +-----+   |
+        // |   | Tab |   |
+        // +---+     +---+
+        //
+        // To get around this we float a border above the tab
+        // so that it'll be 90 degree angle 'butt joint'.
+        //
+        // +---+-----+---+
+        // |   |=====|   |
+        // |   +-----+   |
+        // |   | Tab |   |
+        // +---+     +---+
+        //
+        &:after {
+          content: "";
+          position: absolute;
+          top: -1px;
+          right: 0;
+          left: 0;
+          border-width: 0;
+          border-style: solid;
+          border-color: transparent;
```

Comment:
Totally not blocking but this could be shortened to 
```
border: 0 solid transparent;
```

Reply:
Probably more consistent with the rest of the codebase so I'll change it.